infra/src/cli/generate-chart-values.ts

```diff
@@ -32,7 +34,9 @@ const removeEmptyStringProperties = (obj: any): any => {
 
   return Object.fromEntries(
     Object.entries(obj)
-      .filter(([_, value]) => value !== '') // Filter out empty strings
+      .filter(
+        ([key, value]) => value !== '' && !NON_EMPTYABLE_PROPERTIES.has(key),
+      ) // Filter out empty strings
```

Comment:
Think this condition might be wrong since it is removing non-empty values for SERVERSIDE_FEATURES_ON.

Reply:
Updated the condition, fixed :heart: